tests/test_magnetic_fields.py

```diff
@@ -643,3 +644,37 @@ def test_Bnormal_save_and_load_HELIOTRON(self, tmpdir_factory):
         asym_surf = FourierRZToroidalSurface(sym=False)
         with pytest.raises(AssertionError, match="sym"):
             Bnorm_from_file = read_BNORM_file(path, asym_surf, grid)
+
+    @pytest.mark.unit
+    def test_omnigenous_field_change_well_resolution(self):
+        """Test OmnigenousField.change_resolution() for the magnetic well."""
```

Comment:
this test does not have rtol greater than 3e-3 because you are basically testing points with different monotonic spline interpolations right? so it is not going to be exact like it would be with an object with a spectral basis